pages/[user-id]/my-courses.js

```diff
@@ -2,79 +2,127 @@
 // Conditionally render Student or Mentor View
 import { useEffect, useState } from 'react';
 import { useRouter } from 'next/router';
-import { Grid } from '@mui/material';
+import { Grid, Typography, Stack } from '@mui/material';
+import Link from 'next/link'
 
-import MyCourses from '../../components/MyCourses/MyCourses.js';
-import MyCoursesExampleData from '../../components/MyCourses/data/MyCourses.example.js';
-import { getUserInfo, getCoursesByMenteeId, getCoursesByMentorId } from '../../utils/api/apiCalls.js';
+import {
+  getUserInfo,
+  getCoursesByMenteeId,
+  getCoursesByMentorId,
+  removeCourse,
+  updateCourseInfo,
+  createNewCourse,
+  removeMenteeFromCourse } from '../../utils/api/apiCalls.js';
+import MyCoursesCard from '../../components/MyCourses/MyCoursesCard.js';
+import CreateClassModal from '../../components/CreateClassModal/CreateClassModal.js'
+import MainButton from '../../components/basecomponents/MainButton.js'
+import styles from '../../utils/styles/MyCoursesStyles/MyCourses.module.css';
 
 export default function MyCoursesPage() {
   const router = useRouter();
   var pathUserId = router.asPath.split('/');
   const userID = pathUserId[1];
 
+  const [userInfo, setUserInfo] = useState({ userType: '', id: '', name: { first_name: '', last_name: '' }, avatar_photo: '' })
   const [myCoursesData, setCoursesData] = useState([]);
-  const [userType, setUserType] = useState('');
-
-  const fetchUserInfo = () => {
-    return getUserInfo(userID)
-      .then(res => setUserType(res.account_type))
-      .catch(err => console.log('Error getting user information'))
-  }
 
   const fetchAllCourses = () => {
-    if (userType === 'Mentor') {
+    if (userInfo.userType === 'Mentor') {
       getCoursesByMentorId(userID)
         .then(res => setCoursesData(res))
-        .catch(err => console.log('Error getting course info'))
-    } else if (userType === 'Mentee') {
+        .catch(err => console.warn('Error getting course info'))
+    } else if (userInfo.userType === 'Mentee') {
       getCoursesByMenteeId(userID)
         .then(res => setCoursesData(res))
-        .catch(err => console.log('Error getting course info'))
+        .catch(err => console.warn('Error getting course info'))
+    }
+  }
+
+  const handleDeleteCourse = (courseID) => {
+    removeCourse(courseID)
+      .then(() => fetchAllCourses())
+      .catch(err => console.log('Error deleting course'));
+  }
+
+  const handleStudentDropCourse = (courseID) => {
+    let dropCourseBody = {
+      mentees: {
+        id: userID
+      }
     }
+    removeMenteeFromCourse(courseID, dropCourseBody)
+      .then(() => fetchAllCourses())
```

Comment:
If we aren't passing any args into the .then callback function, we dont need to wrap it in an arrow func. Could do 
`.then(fetchAllCourses).catch(...)`

Reply:
👍